model/textparse/benchmark_test.go

```diff
@@ -78,6 +82,12 @@ func BenchmarkParse(b *testing.B) {
 		// We don't pass compareToExpfmtFormat: expfmt.TypeOpenMetrics as expfmt does not support OM exemplars, see https://github.com/prometheus/common/issues/703.
 		{dataFile: "omtestdata.txt", parser: "omtext"},
 		{dataFile: "promtestdata.txt", parser: "omtext"}, // Compare how omtext parser deals with Prometheus text format vs promtext.
+
+		// NHCB.
+		{dataFile: "omhistogramdata.txt", parser: "omtext"},           // Measure OM parser baseline for histograms.
+		{dataFile: "omhistogramdata.txt", parser: "nhcb_over_omtext"}, // Measure NHCB over OM parser.
+		{dataFile: "omcounterdata.txt", parser: "omtext"},             // Measure OM parser baseline for counters.
```

Comment:
This file is so small and only have some counters. Do we need this? We already bench this `omtestdata.txt` and `promtestdata.txt`? Anyone can create such benchmark for themselves, but it feels unrealistic?

Reply:
Removed. That's fair, but I'd like to keep omhistogramdata.txt to check performance on histograms directly.